main/scala/scalangband/ui/NewGameWindow.scala

```diff
@@ -7,41 +7,55 @@ import scalangband.model.settings.Settings
 import scalangband.model.{Game, Player}
 
 import scala.swing.event.ButtonClicked
-import scala.swing.*
+import scala.swing._
```

Comment:
Is there a reason this needed to change to `._`? I think `.*` is the preferred Scala 3 format.

Reply:
hmm let me check on this after work, i had some problem with intellij and had to mess around with the imports.
let me check later if i'm able to use the imports from the original repo